tests/utils.py

```diff
@@ -2,18 +2,26 @@
 import dcos.http
 import dcos.package
 
-import json
 import logging
 import os
 import re
-import requests
 import shakedown
 import subprocess
 import urllib
+import urllib.parse
```

Comment:
is there a reason this can't be done with the above import?

Reply:
If I only import `urllib`, the following line will fail:
`app_id_encoded = urllib.parse.quote(app_id, safe='')`